test/parallel/test-http-write-head.js

```diff
@@ -18,6 +18,17 @@ var s = http.createServer(function(req, res) {
   }
   assert.ok(threw, 'Non-string names should throw');
 
+  // undefined value should throw, via 979d0ca8
+  threw = false;
+  try {
```

Comment:
IMO it would make more sense to use `assert.throws()` with a validation function like:

```javascript
function(err) {
  return err instanceof Error && err.message === '`value` required in setHeader("foo", value).';
}
```